pkg/compiler/testdata/errors2.txt

```diff
@@ -123,12 +123,12 @@ s8 {
 } [align[7]]			###  bad struct s8 alignment 7 (must be a sane power of 2)
 
 u0 [
-	f	len[f1, int32]	### len target f1 does not exist
+	f	len[f1, int32]	### len target f1 does not exist in u0, parents: <foo$202>, s7
```

Comment:
Is the list of parents useful?
I am trying to understand when it would be useful for me, but don't see yet. We print the struct/syscall name where we are looking for the field. I can see it may be useful, esp for "syscall". But I am not sure about parents.

Reply:
Hmm, indeed. I've left the `in %s` part, but dropped `parents:`.